### PR TITLE
Lvnodcdxi 43

### PR DESCRIPTION
**Pull request by StanislavFedyna**
- Make reusable component - button
- Deleted sound Modes / additional Config components

### CODE REVIEW

File: src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.css [hunk 1]
```diff
@@ -1,35 +1,70 @@
...
+  text-align: right
 }
 
-.textArea {
-padding-left: 20px;
+.headerModalWindow .icon {
+ margin: 10px;
+}
+
+.textArea strong {
```
Comment (GeFeSTTS) [inline src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.css]: Don't chage styles in standart styled tags. Use span, for example.

**Review by MaxVoloskiy — DISMISSED** [review]

**Review by mr-redFox — DISMISSED** [review]

File: src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.css (hunk 1 above)
Comment (StanislavFedyna) [reply src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.css]: Done)

**Review by GeFeSTTS — DISMISSED** [review]

**Comment by tonypntsrn** [comment]
Pull request’s description should contain a brief of what have been done, so which problems did you fixed?

File: src/client/components/equalizer/equalizer.js [hunk 2]
```diff
@@ -182,31 +184,54 @@ class Equalizer extends Component {
       uploadSoundInfoFromFile,
       setCanvasToState,
     } = this;
+    const { audioData } = this.props;
```
Comment (tonypntsrn) [inline src/client/components/equalizer/equalizer.js]: You can destruct everything from audio data right here, no need to create another variable, if you don’t use audio data itself

File: src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js [hunk 3]
```diff
@@ -1,32 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './infoAboutUploadFile.css';
+import { musicIcon } from '../../../../assets/icons/icons';
 
 const InfoAboutTrack = (props) => {
-  const { trackname, tracksize, tracktype } = props;
+  const { trackname } = props;
```
Comment (tonypntsrn) [inline src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js]: You can destruct this, right from function parameters

File: src/client/components/componentWithSliders/componentWithSliders.css [hunk 4]
```diff
@@ -25,3 +25,7 @@
   display: flex;
   justify-content: space-around;
 }
+
+.SlidersComponent .SavePresetButton {
+  margin-left: 0px;
```
Comment (tonypntsrn) [inline src/client/components/componentWithSliders/componentWithSliders.css]: If smth is 0 in css, no need to specify which units you are using, if it’s px, rem or etc, it’s still be 0

Comment (StanislavFedyna) [reply src/client/components/componentWithSliders/componentWithSliders.css]: done)

File: src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js (hunk 3 above)
Comment (StanislavFedyna) [reply src/client/components/equalizer/upload/infoAboutFile/infoAboutUploadFile.js]: done)

File: src/client/components/equalizer/equalizer.js (hunk 2 above)
Comment (StanislavFedyna) [reply src/client/components/equalizer/equalizer.js]: done)

**Review by MaxVoloskiy — APPROVED** [review]

**Review by GeFeSTTS — APPROVED** [review]

**Pull request merged**